packages/gatsby-theme-bodiless/create-node.js

```diff
@@ -61,6 +66,30 @@ const findFilesystemNode = ({ node, getNode }) => {
   return fsNode;
 };
 
+/**
+ * Get current git repo info.
+ *
+ * @returns Promise<{
+ *  repo: string,
+ *  hash: string,
+ *  branch: string,
+ * }>
+ */
+const createGitInfo = async () => {
+  const gitDir = await findGitFolder();
```

Comment:
What happens if Bodiless is deployed without using git? For instance, a CI/CD system might generate a build and deploy it directly into a server, so the .git folder might not be included. `gitDir` could be undefined, or a path to a git repo not related to Bodiless found somewhere up in the file system.

Should this code account for these cases?

Reply:
updated